lib/OpenQA/WebAPI/Controller/Main.pm

```diff
@@ -108,6 +108,7 @@ sub _group_overview ($self, $resultset, $template) {
     my $page       = $validation->param('comments_page')  // 1;
     my $page_limit = $validation->param('comments_limit') // 5;
 
+    $self->inactivity_timeout($ENV{OPENQA_WEBUI_INACTIVITY_TIMEOUT} // 90);
```

Comment:
The name of the environment variable should likely be less generic as we only want to affect this single route¹.

```suggestion
    $self->inactivity_timeout($ENV{OPENQA_WEBUI_OVERVIEW_INACTIVITY_TIMEOUT} // 90);
```

¹ Otherwise one should be able to just set `MOJO_INACTIVITY_TIMEOUT` which would affect all routes without explicit overrides like this one.